eox_nelp/signals/utils.py

```diff
@@ -44,7 +44,7 @@ def _generate_external_certificate_data(time, certificate_data):
         "id": certificate.id,
         "created_at": time.date(),
         "expiration_date": None,
-        "grade": int(certificate_data.grade),
+        "grade": float(certificate_data.grade) * 100,
```

Comment:
Does certificates receive float ? or This have to be int? Or could be float like 87.8?

Reply:
This field was set a CharField, and  the current prod implementation shows values between  0 and 1 where  1 is  100% and  0 is 0% so there are values like 0.85